### DISCUSSION TITLE
EnergyAPI Bridge

### DISCUSSION THREAD

**Discussion by MSandro** [Sinytra/ForgifiedFabricAPI#48] [Ideas]
Hello. First of all, I would like to express my utmost respect. I'm really impressed with what already works with the connector and Forgified.

![image](https://github.com/Sinytra/ForgifiedFabricAPI/assets/10937159/be8261cc-1c38-4dd4-8e0b-2d2220cbc21a)
Is there a plan to implement an energy bridge?
It would be great if Fabric energy sources could fill the machines from Forge and vice versa.

**Comment by Su5eD** [comment]
Hi, an energy bridge for FE <-> TR E is now available in [Connector Extras](https://github.com/Sinytra/ConnectorExtras), enabled by default, with configurable conversion ratios for both ways.

**Comment by MSandro** [comment]
Wow amazing 👍👍
